contracts/adder/lib.rs

```diff
@@ -0,0 +1,65 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+//! This is a simple example contract for use with e2e tests of the aleph-client contract interaction.
+
+#[ink::contract]
+mod adder {
+    #[ink(storage)]
+    pub struct Adder {
+        name: Option<[u8; 20]>,
```

Comment:
[nit]: There's a `String` [type in ink](https://docs.rs/ink_prelude/latest/ink_prelude/string/struct.String.html). Your choice.

Reply:
But that one won't have constant-sized storage, will it? Can it even be used as args/return values of contract methods because of that?